tests/incr_test.go

```diff
@@ -0,0 +1,43 @@
+package tests
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+)
+
+type tcase struct {
+	op  string
+	val int64
+}
+
+func TestINCR(t *testing.T) {
+	var wg sync.WaitGroup
+	go runTestServer(&wg)
+	conn := getLocalConnection()
+
+	for _, tc := range []tcase{
+		{"s1", 0},
+		{"i1", 1},
+		{"i1", 1},
```

Comment:
why `s1` and `i1`, you would have used `s` and `i` itself.

Reply:
what about test case when key does not exist and we directly increment?

Reply:
You can define the

```
type tcase struct {
	op  string
	val int64
}
```

as 

```
type tcase struct {
	op  string
	key string
	val int64
}
```

and then make the implementation neater.

Reply:
done the changes